src/app/[locale]/commerce/settings/page.tsx

```
@@ -0,0 +1,573 @@
+                  </select>
+                </div>
+                <div>
+                  <label className="block text-sm font-medium text-gray-700 mb-2">
+                    Decimal Places
+                  </label>
+                  <input
+                    type="number"
+                    min="0"
+                    max="4"
+                    value={settings.currency.decimalPlaces}
+                    onChange={(e) => updateSettings('currency', 'decimalPlaces', parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)